tests/factories/incident.py

```diff
@@ -0,0 +1,42 @@
+import random
+
+from django.db.models.signals import post_save
+import factory
+from faker import Factory
+
+from response.core.models import Incident, ExternalUser
+
+faker = Factory.create()
+
+
+@factory.django.mute_signals(post_save)
+class IncidentFactory(factory.DjangoModelFactory):
```

Comment:
Use [factory_boy](https://factoryboy.readthedocs.io/en/latest/) to easily generate random test data 🙌